### ISSUE TITLE
Missing type for placeholder on Select.Trigger

### ISSUE THREAD

**Issue by martinsvatek** [radix-ui/themes#185]
Missing type for placeholder on Select.Trigger:
<img width="1152" alt="image" src="https://github.com/radix-ui/themes/assets/48355212/b36bc697-0cae-4195-acb1-c63e83cdca95">

Docs:
<img width="755" alt="image" src="https://github.com/radix-ui/themes/assets/48355212/6fe36eaa-e283-4615-829d-8d580b2516b5">

Only type check is a problem, usage and everything else is working correctly :)

**Comment by vladmoroz** [comment]
Hmm works for me, what’s your TypeScript version? 

<img width="596" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/767875eb-bc77-4330-aa11-633cd36cc4bc">

***

<img width="477" alt="image" src="https://github.com/radix-ui/themes/assets/8441036/dc9db316-1c94-47a2-80bb-7c9defef4e3a">

**Comment by martinsvatek** [comment]
Hello, thanks for your reply :) `"typescript": "^5.3.3"`, `"@radix-ui/themes": "^2.0.2",`

**Comment by noinskit** [comment]
I'm hitting this problem as well.

It's an interaction with a recent change in `@types/react`:
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/b954269038de46b4b2f1756a9f2f020cbc66a326
`@types/react` 18.2.42 works for me, 18.2.45 does not; I haven't checked versions in between.

**Comment by alaindeurveilher** [comment]
Could we have a quick patch release with this fix please? as our build now fail without the fix. Thank you.

**Comment by alaindeurveilher** [comment]
Actually, build can pass temporarily by ignoring typing:
```tsx
{/* @ts-ignore */}
<Select.Trigger placeholder={placeholder} />
```

**Comment by vladmoroz** [comment]
Hey, just released 2.0.3 with this fix
